epictrack-web/src/components/myWorkplans/Card/CardBody.tsx

```diff
@@ -0,0 +1,149 @@
+import { Box, Grid, SxProps } from "@mui/material";
+import { Palette } from "../../../styles/theme";
+import { ETCaption1, ETCaption2, ETHeading4, ETParagraph } from "../../shared";
+import Icons from "../../icons";
+import { IconProps } from "../../icons/type";
+
+const IndicatorSmallIcon: React.FC<IconProps> = Icons["IndicatorSmallIcon"];
+const DotIcon: React.FC<IconProps> = Icons["DotIcon"];
+const ClockIcon: React.FC<IconProps> = Icons["ClockIcon"];
+
+const CardBody = () => {
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="flex-start"
+      sx={{
+        backgroundColor: Palette.white,
+        padding: "16px 24px",
+      }}
+      gap={1}
+    >
+      <Grid item container direction="row" spacing={1}>
+        <Grid item>
+          <ETHeading4 bold color={Palette.neutral.dark}>
+            EAC Assessment
+          </ETHeading4>
+        </Grid>
+        <Grid item>
+          <ETCaption1
+            bold
+            sx={{
+              color: Palette.success.dark,
+              backgroundColor: Palette.success.bg.light,
+              padding: "4px 8px",
+              borderRadius: "4px",
+            }}
+          >
+            In Progress
+          </ETCaption1>
+        </Grid>
+      </Grid>
+      <Grid
+        item
+        container
+        direction="row"
+        spacing={1}
+        sx={{ paddingBottom: "8px" }}
+      >
+        <Grid item sx={{ marginTop: "2px" }}>
+          <DotIcon />
+        </Grid>
+        <Grid item>
+          <ETCaption2
+            bold
+            color={Palette.neutral.main}
+            sx={{ fontSize: "14px" }}
```

Comment:
when it comes to text properties like fontSize or lineHeight we shouldn't do spot changes for them. There should be a text with fontSize 14px in the shared components if there isn't we should talk to Maria and create one